src/workerd/api/sockets.c++

```diff
@@ -28,7 +28,6 @@ bool isValidAddress(kj::StringPtr address) {
       case ':':
       case '_':
       case '[': case ']': // For IPv6.
-      case '/': // To enable "unix:/" addresses (only used for testing, the proxy should reject this).
```

Comment:
alternatively, we could probably keep the case but use something like the following the catch the "only used for testing" mode:

```
  case '/': {
    if (isPredictableModeForTest()) {
     breakl
    }
  }
```

Reply:
The tests don't have any need for this anymore.